docs/pages/introduction/why-not-expo.mdx

```diff
@@ -3,108 +3,110 @@ title: Limitations
+Apps built with `expo build:ios|android` contain code for the entire Expo SDK. Unfortunately, you cannot customize the native dependencies, including Facebook's Audience Network library. So if you build your app this way, you cannot designate it as "designed primarily for children under 13" in the app stores, even though this code is never run unless you explicitly call it.
 
 </Collapsible>
 
-> We are either actively working on or planning to build solutions to all of the limitations listed above, and if you think anything is missing, please bring it to our attention by posting to our [feature requests board](https://expo.canny.io/feature-requests) or the [forums](https://forums.expo.dev/).
+## Bare workflow
 
-## Limitations of the bare workflow
+> If you'd like to continue getting the same development experience that you have with Expo Go, but also use custom native libraries or custom native code, you should check out the [Development builds]()
 
-In the bare workflow, we have full access to the underlying native projects and any native code. It's a "bare" native project with React Native and one or more packages from the Expo SDK installed. Anything that you can do in a native project is possible here.
+As similar to [Development builds], bare workflow also provides access to the underlying native projects and any native code. It's a "bare" native project where developers make direct changes to their native `android` and `ios` project directories rather than continuously generating them on demand using the [Expo config (**app.json**) and prebuild](/workflow/prebuild/).
 
-The following list is therefore specifically oriented towards the limitations that exist around using Expo tools and services in the bare workflow.
+The following list is specifically oriented towards the limitations that exist around using Expo tools and services in the bare workflow.
 
-<Collapsible summary={<span><span className="strike">Build service only works in the managed workflow</span> (✅ Resolved in December 2020)</span>}>
+<Collapsible summary={<span><span className="strike">Build service only works in the managed workflow</span> (Resolved in December 2020)</span>}>
 
 ~~To build your app binaries for distribution on the Apple App Store and Google Play Store you will need to follow the same steps that you would in any native project, the Expo build service can't handle it for you. We are working on bringing bare workflow support to the build service in the near future.~~
 
-You can now use [EAS Build](/build/introduction) to build and sign your apps just as easily as in the managed workflow! [Read the announcement blog post](https://blog.expo.dev/expo-application-services-eas-build-and-submit-fc1d1476aa2e).
+You can now use [EAS Build](/build/introduction) to build and sign your apps. [Read the announcement blog post](https://blog.expo.dev/expo-application-services-eas-build-and-submit-fc1d1476aa2e).
```

Comment:
```suggestion
```
I'd be in favor of striking anything that suggests we just changed this yesterday.